### PR TITLE
Review one of your past projects

### PR DESCRIPTION
**Pull request by maximoortelli**

### CODE REVIEW

**Comment by maximoortelli** [comment]
### ESTADO: Cambios requeridos ♻️♻️♻️

Hola @maximoortelli

¡Buen trabajo hasta ahora!
Hay algunos problemas en los que aún debe trabajar para pasar al siguiente proyecto, ¡pero ya casi está allí!

161814298-b87179c3-315f-4207-ac3b-1d3e584d506e

### Reflejos:
✅ Sin error de filtro
✅ LÉAME es profesional

### Cambios requeridos ♻️

Buen trabajo hasta ahora, su código se ve muy bien, sin embargo, en el archivo README.md aparece que su nombre de proyecto es "Portfolio: Finish mobile version" cuando en realidad es "Review one of your past projects" por favor realice los siguientes cambios para mejorar el codigo.

### Sugerencias opcionales

Cada comentario con el prefijo [OPCIONAL] no es lo suficientemente importante como para detener la aprobación de este PR. Sin embargo, le recomiendo encarecidamente que los tenga en cuenta, ya que pueden mejorar su código.

Saludos y feliz codificación!👏👏👏

Siéntase libre de dejar cualquier pregunta o comentario en el hilo de relaciones públicas si algo no está 100% claro.
Por favor, recuerda etiquetarme en tu pregunta para que pueda recibir la notificación.

Por favor, no abra una nueva solicitud de extracción para nuevas revisiones. Debe usar la misma Solicitud de extracción enviada para la primera revisión, ya sea válida o no válida, a menos que se solicite lo contrario.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="cat-black-face.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio: finish mobile version</b></h3>
+
```
Comment (maximoortelli) [inline README.md:8]: - Please make sure you make the changes to the README.md file correctly, it is very important that your project is properly named.

Comment (Olamarx) [reply README.md:8]: > * Please make sure you make the changes to the README.md file correctly, it is very important that your project is properly named.

- [x] Good job making a polite comment but I would suggest you make a clear reason for the changes, while it being "important" is good but it is not sufficient for the student to understand. The question to answer here is "Why is it important?"

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @maximoortelli, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Polite comment ✔️
- Inline review ✔️
- GitHub review features usage ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] I would implore you to make at least two inline comments as required in the project requirements.
image

- [ ] I would advise you to use the English language too for your review summary, not only because English is the Microverse accepted way of communication, but because English is the most used language in technology all over the internet around the world.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by maximoortelli** [comment]
### **STATUS: Changes required ♻️♻️♻️**

Hello @maximoortelli
Good job so far!
There are a few issues you still need to work on to move on to the next project, but you're almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966 (1)


### **Highlights:**

- Polite comment ✔️

- Inline review ✔️

- GitHub review features usage ✔️

### **Required Changes ♻️**
_Check the comments under the review._

- [ ]  I would implore you to rename the project "Portfolio - Finish Mobile" to "Review one of your past projects" in your README.md file to improve your project.

- [ ]   I would advise you to change the URL of your git clone in the README.md file "https://github.com/maximoortelli/Portfolio-setup-and-mobile-first.git" to "https://github.com/maximoortelli/Project-review.git" to successfully clone the project.

### **Optional suggestions**
_Each comment prefixed with [OPTIONAL] is not important enough to stop this PR from being approved. However, I strongly recommend that you keep them in mind, as they can improve your code._

Cheers and happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can get notified.

**Please do not open a new pull request for new revisions. You must use the same Pull Request submitted for the first review, whether valid or invalid, unless otherwise requested.**

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @maximoortelli , 



Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
- [ ] I noticed that in making the changes required by the last reviewer you failed to add your comments as inline comments rather you made the comments in the review summary. Please i will implore you to try exploring the use of inline comments. since the changes you are requesting are in the README.md file you can go to that specific line in the readme so the student is fully aware what he need to change. please can you kindly add these changes before resubmiting for another review. Thank you :+1: :sparkles: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="cat-black-face.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio: finish mobile version</b></h3>
```
Comment (maximoortelli) [inline README.md:7]: - This change is important to make to get a professional readme file for the future.👍🚀
- Please Make sure you make the changes in the README.md file correctly in the h3 tag inside you put "Portfolio: finish mobile version" please change it to "Review one of your past projects" it is very important that your project has the correct name. 😃🖥

File: README.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+2. A Web Browser with developer tools that supports HTML5 and CSS.
+
+### Setup
+1. On GitHub.com, navigate to the main page of the repository.
+2. Above the list of files, click *Code*.
+3. Copy the URL for the repository.
+4. Open Terminal.
+5. Change the current working directory to the location where you want the cloned directory.
+6. Type git clone, and then paste the URL you of the reposotory.<br>
+``` $ git clone https://github.com/maximoortelli/Portfolio-setup-and-mobile-first.git ```
```
Comment (maximoortelli) [inline README.md:85]: - Kindly make sure to remove this placeholder and add your repository link for cloning purposes.
- You currently do not have a profesional README file, however this is an important requirement in future proyects you will build here at Microverse 👍
- I would advise to resolve this, change the URL of your git clone in the README.md file "https://github.com/maximoortelli/Portfolio-setup-and-mobile-first.git" to "https://github.com/maximoortelli/Project-review.git" to successfully clone the project.👍😃

**Review by maximoortelli — COMMENTED** [review]
### **STATUS: Changes required ♻️♻️♻️**
Hello @maximoortelli
Good job so far!
There are a few issues you still need to work on to move on to the next project, but you're almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

### **Highlights:**
Polite comment ✔️

Inline review ✔️

GitHub review features usage ✔️

### **Required Changes ♻️**
Check the comments under the review.

- [ ] Please Make sure you make the changes in the README.md file correctly in the h3 tag inside you put "Portfolio: finish mobile version" please change it to "Review one of your past projects" it is very important that your project has the correct name. 😃

- [ ] You currently do not have a profesional README file, however this is an important requirement in future proyects you will build here at Microverse.

- [ ] I would advise to resolve this, change the URL of your git clone in the README.md file "https://github.com/maximoortelli/Portfolio-setup-and-mobile-first.git" to "https://github.com/maximoortelli/Project-review.git" to successfully clone the project.👍😃

### **Optional suggestions**
Each comment prefixed with [OPTIONAL] is not important enough to stop this PR from being approved. However, I strongly recommend that you keep them in mind, as they can improve your code.

Cheers and happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can get notified.

Please do not open a new pull request for new revisions. You must use the same Pull Request submitted for the first review, whether valid or invalid, unless otherwise requested.

**Review by maximoortelli — COMMENTED** [review]
Changes made successfully.

**Review by Shisui6 — APPROVED** [review]
Hi @maximoortelli, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**